src/layout/section/illustratedSection/index.tsx

```diff
@@ -0,0 +1,41 @@
+import styled from 'styled-components'
+import { responsiveBreakpoints, space } from '_utils/branding'
+
+import IllustratedSection from './illustratedSection'
+
+const StyledIllustratedSection = styled(IllustratedSection)`
+  & .section-content {
+    display: flex;
+    flex-direction: column;
+  }
+
+  & .kirk-illustratedSection-illustration {
+    margin-left: -${space.xl};
```

Comment:
Why do we need these negative margins ?
Does it mean the illustration is touching the viewport edges in mobile mode ?

I would probably remove these to make it simpler (and the missing padding probably does not really matter)

Reply:
The illustration touches the borders of the screen on mobile, yes.